source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+   4. If we chose to filter the data to retain only samples with 2500 sequences, how many samples would we lose?
+   5. Which features are observed in at least 47 samples?
+
+Generating a Phylogenetic Tree for Diversity Analysis
+=====================================================
+
+QIIME 2 analysis allows the use of phylogenetic trees for both diversity metrics such as PD whole tree and UniFrac distance as well as feature-based analyses in Gneiss. The tree provides an inherent structure to the data, allowing us to consider an evolutionary relationship between organisms.
+
+QIIME 2 offers several ways to construct a phylogenetic tree. For this tutorial, we’re going to use a fragment insertion tree using the ``fragment-insertion`` plugin. The authors of the fragment insertion plugin suggest that it can outperform traditional alignment based methods based on short illumina reads by alignment against a reference tree built out of larger sequences. Our command, ``qiime fragment-insertion sepp`` will take the representative sequences (a ``FeatureData[Sequence]`` object) we generated during deblurring and return a phylogenetic tree where the sequences have been inserted into the greengenes 13_8 99% identity reference tree backbone.
+
+*Note: This command tables about 11 minutes to run.*
+
+.. command-block::
+
+   qiime fragment-insertion sepp \
+     --i-representative-sequences ./deblur_rep_set.qza \
+     --o-tree ./tree.qza \
+     --o-placements ./tree_placements.qza
+
+
+Taxonomic Classification
+========================
+
+Let’s do one more preparation step before we dig into the analysis! To be able to identify ASVs and give them “names”, we need to somehow determine taxonomy. To do this, we’ll use the ``q2-feature-classifier`` plugin.
+
+For this analysis, we'll use a pretrained classifier using 99% Greengenes 13_8 reference set trimmed to 250 bp of the V4 hypervariable region (corresponding to the 515F-806R primers). The classifier is a specific semantic type, ``TaxonomicClassifier``, and it is actually the object that does the classification.
+
+.. download::
+   :url: https://data.qiime2.org/2019.4/common/gg-13-8-99-515-806-nb-classifier.qza
+   :saveas: gg-13-8-99-515-806-nb-classifier.qza
+
+It’s worth noting that naive bayesian classifiers perform best when they’re trained for the specific hypervariable region amplified. You can train a classifier specific for your dataset based on the :doc:`training classifiers tutorial <feature-classifier>` or download classifiers for other datasets from the :doc:`QIIME 2 resource page <../data-resources>`. Classifiers can be re-used for consistent versions of the underlying packages, database and region of interest.
+
+.. command-block::
+
+   qiime feature-classifier classify-sklearn \
+     --i-reads ./deblur_rep_set.qza \
+     --i-classifier ./gg-13-8-99-515-806-nb-classifier.qza \
+     --o-classification ./taxonomy.qza
+
+.. do we want to throw clawback in here?
```

Comment:
@nbokulich - thoughts?

Reply:
_maybe_... for the purposes of the workshop it would be neat to showcase clawback, but for the purposes of a general-use tutorial clawback may be a bit complicated (multiple additional analysis steps, etc).

Reply:
I vote for skipping ATM, let's focus on Copenhagen as our initial target.

Reply:
Im happy either way